src/app.js

```diff
@@ -42,20 +44,27 @@ module.exports = class App {
 
         // TODO: i prefer to check it before inject here!
         // let context = {
+        //     id: '',
         //     input: '',
         //     handle: () => {}
         // }
 
+        let user = {};
+
+        if (context.id) {
+            user = await db.getUser(context.id);
+        }
 
         response = await this._execute(this.modules, response, {
             ...this.context,
```

Comment:
`this.context` and `context` looks meeeh

Reply:
agree, but dont know how implement it by better way